README.adoc

```diff
@@ -131,6 +131,24 @@ This can be achived with the {uri-kube-apiserver}[Kubernetes API server CLI], e.
 $ kube-apiserver --cors-allowed-origins .*
 ```
 
+== Authentication
+
+We try to support the various authentication strategies supported by https://kubernetes.io/docs/reference/kubectl/overview/[kubectl] in order to provide seamless integration with your local setup. Here are the different authentication strategies we support depending on how you're using Kubebox:
+
+[cols="<,^,^,^,^,^", options="header"]
+|===
+||Executable |Server |Kubectl | Docker| Online
+
+|OpenID Connect | icon:check[] | icon:check[] | icon:check[] | icon:check[] | icon:check[]
```

Comment:
I would add a table note mentioning that custom IDP certificate authority file is not supported in Web versions.

Reply:
Do you think that's worth adding for the online mode?

Reply:
For sure! Added a note